RFCs/0011-DeletionPolicy.md

```diff
@@ -0,0 +1,262 @@
+# Allow Intrinsic Functions and Pseudo-Parameter References in `DeletionPolicy` and  `UpdateReplacePolicy` Resource attributes
+
+* **Original Author(s):**: @mingujo
+* **Tracking Issue**: [Tracking Issue](https://github.com/aws-cloudformation/cfn-language-discussion/issues/11)
+
+# Summary
+
+Use CloudFormation [intrinsic functions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference.html) and [Pseudo-Parameter references](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/pseudo-parameter-reference.html) to dynamically determine a value for DeletionPolicy and UpdateReplacePolicy Resource attributes
+
+# Examples
+
+### Use [Ref](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-ref.html)
+```
+{
+    "AWSTemplateFormatVersion" : "2010-09-09",
+    "Parameters": {
+        "DeletionPolicyParam": {
+            "Type": "String",
+            "AllowedValues": ["Delete", "Retain", "Snapshot"],
+            "Default": "Delete"
+        },
+        "UpdateReplacePolicyParam": {
+            "Type": "String",
+            "AllowedValues": ["Delete", "Retain", "Snapshot"],
+            "Default": "Delete"
+        }
+    },
+    "Resources": {
+        "Table": {
+            "Type": "AWS::DynamoDB::Table",
+            "Properties": {
+              "KeySchema": [
+               {
+                 "AttributeName" : "primaryKey",
+                 "KeyType" : HASH
+               } 
+              ]
+            },
+            "DeletionPolicy": { "Ref": "DeletionPolicyParameter" },
+            "UpdateReplacePolicy": { "Ref": "UpdateReplacePolicyParam" }
+        }
+    }
+}
+```
+
+### Use [Conditions](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/conditions-section-structure.html) and [Fn::If](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-condition.html) based on Parameters
```

Comment:
Shouldn't the Fn::If link go to this URL? https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html#intrinsic-function-reference-conditions-if